tests/unit/CRUD/base/base-reading-true-predicates-test.js

```diff
@@ -0,0 +1,57 @@
+import Ember from 'ember';
+import QueryBuilder from 'ember-flexberry-data/query/builder';
+import { TruePredicate } from 'ember-flexberry-data/query/predicate';
+
+export default function readingPredicatesTruePredicates(store, assert) {
+  assert.expect(3);
+  let done = assert.async();
+
+  Ember.run(() => {
+    initTestData(store)
+
+    // Contains.
+    .then(() => {
+      let builder = new QueryBuilder(store, 'ember-flexberry-dummy-application-user');
+
+      return store.query('ember-flexberry-dummy-application-user', builder.build())
+      .then((data) => {
+        assert.ok(data.every(item => item.get('name') != null), 'Contains with correct data | Data');
```

Comment:
Почему у нас линтер нестрогое сравнение пропускает...
```suggestion
        assert.ok(data.every(item => item.get('name') !== null), 'Contains with correct data | Data');
```